### PR TITLE
chore(docs): document query cache config props

### PR DESCRIPTION
**Pull request by puzpuzpuz**
A follow-up for https://github.com/questdb/questdb/pull/1735

Also adds a note on the SELECT query cache to the Grafana tutorial

### CODE REVIEW

File: docs/operations/backup.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ curl -G --data-urlencode "query=BACKUP database;" \
 :::caution
 
 To run a reliable filesystem backup, the QuestDB instance must be shut down
-**or** no write operations should running while disk backup is being created.
+**or** no write operations should running be while disk backup is being created.
```
Comment (bsmth) [inline docs/operations/backup.md:80]: typo "should be running"

File: tutorial/2020-10-19-grafana.mdx [hunk 2]
```diff
@@ -123,6 +123,11 @@ setting the property `http.security.readonly=true` in your `server.conf`.
 Details of setting this configuration can be found on QuestDB's
 [configuration page](/docs/reference/configuration/).
 
+To avoid unnecessary memory usage, it is also recommended to disable SELECT query
+cache by setting the property `pg.select.cache.enabled=true` in your `server.conf`.
+That's because Grafana does not use prepared statements when sending the queries
+and the query cache becomes much less efficient.
```
Comment (bsmth) [inline tutorial/2020-10-19-grafana.mdx:129]: This is a good hint. Could we also add this to https://github.com/questdb/questdb.io/blob/master/docs/third-party-tools/grafana.md ?

File: docs/operations/backup.md (hunk 1 above)
Comment (puzpuzpuz) [reply docs/operations/backup.md]: Fixed in 9868c6b29b2c5673c862d3869e045d2d737dc65a

File: docs/reference/configuration.md [hunk 3]
```diff
@@ -236,6 +236,9 @@ default on port `9000`. For details on the use of this component, refer to the
...
 | http.json.query.double.scale                 | 12             | The scale value of string representation of `DOUBLE` values                                                                                                                                                                                                                                                                                                                                                                        |
+| http.query.cache.enabled                     | true           | Enable or disable the query cache. Cache capacity is number_of_blocks \* number_of_rows.                                                                                                                                                                                                                                                                                                                                           |
```
Comment (bsmth) [inline docs/reference/configuration.md:239]: Maybe it's more readable to format in monospace:

```
Cache capacity is `number_of_blocks * number_of_rows`.
```

File: tutorial/2020-10-19-grafana.mdx (hunk 2 above)
Comment (puzpuzpuz) [reply tutorial/2020-10-19-grafana.mdx:129]: Done in da66859a899b1a3dd896cfcc205caa3e6c80501f

File: docs/reference/configuration.md [hunk 4]
```diff
@@ -347,8 +348,12 @@ PostreSQL wire protocol.
...
-| pg.factory.cache.column.count       | 16           | Number of columns to cache SQL execution plan against text to speed up execution. This value must be 2n (power of two).                                                                                                                                                                     |
-| pg.factory.cache.row.count          | 16           | Number of rows to cache for SQL execution plan against text to speed up execution. This value must be 2n (power of two)                                                                                                                                                                     |
+| pg.select.cache.enabled             | true         | Enable or disable the SELECT query cache. Cache capacity is number_of_blocks \* number_of_rows.                                                                                                                                                                                             |
```
Comment (bsmth) [inline docs/reference/configuration.md:351]: See comment above

File: docs/reference/configuration.md [hunk 5]
```diff
@@ -347,8 +348,12 @@ PostreSQL wire protocol.
...
+| pg.select.cache.block.count         | 16           | Number of blocks to cache SELECT query execution plan against text to speed up execution.                                                                                                                                                                                                   |
+| pg.select.cache.row.count           | 16           | Number of rows to cache for SELECT query execution plan against text to speed up execution.                                                                                                                                                                                                 |
+| pg.insert.cache.enabled             | true         | Enable or disable the INSERT query cache. Cache capacity is number_of_blocks \* number_of_rows.                                                                                                                                                                                             |
```
Comment (bsmth) [inline docs/reference/configuration.md:354]: Same as above

File: tutorial/2020-10-19-grafana.mdx (hunk 2 above)
Comment (bsmth) [reply tutorial/2020-10-19-grafana.mdx:129]: Thanks a lot

File: docs/reference/configuration.md (hunk 3 above)
Comment (puzpuzpuz) [reply docs/reference/configuration.md]: Applied the suggestion in 00b68b99fb9e5082b4dde21173d3fc9ce70fc758

**Review by bsmth — APPROVED** [review]
Thanks for the additions! 👍🏻

**Pull request merged**